.travis.yml

```diff
@@ -0,0 +1,50 @@
+# We set the language to c because python isn't supported on the MacOS X nodes
+# on Travis. However, the language ends up being irrelevant anyway, since we
+# install Python ourselves using conda.
+language: c
+
+os:
+    - linux
+
+# Use Travis' container-based architecture
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - graphviz
+    - texlive-latex-extra
+    - dvipng
+
+env:
+    global:
+        # The following versions are the 'default' for tests, unless
+        # overidden underneath. They are defined here in order to save having
+        # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - PYTEST_VERSION=3.2
+        - NUMPY_VERSION=stable
+        - ASTROPY_VERSION=stable
+        - CONDA_DEPENDENCIES='six'
+
+    matrix:
+        - PYTHON_VERSION=2.7 PYTEST_VERSION=2.8
+        - PYTHON_VERSION=2.7
+        - PYTHON_VERSION=3.5 PYTEST_VERSION=3.0
+        - PYTHON_VERSION=3.6 PYTEST_VERSION=3.1
+        - PYTHON_VERSION=3.6
+
+matrix:
+    include:
+        # Try a run on OSX with latest versions of python and pytest
+        - os: osx
+
+install:
+    - git clone git://github.com/astropy/ci-helpers.git
+    - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
```

Comment:
please use the generic version, ``setup_conda.sh``

Reply:
Do you think this is the cause of the issue with pytest above?

Reply:
nope, this will just make sure we can use the few workarounds that are included in the generic file (e.g. ``skip travis``, or ``docs only``, or cron, etc.)